go/kbfs/search/mapping.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package search
+
+import (
+	"github.com/blevesearch/bleve"
+	"github.com/blevesearch/bleve/analysis"
+	"github.com/blevesearch/bleve/analysis/char/html"
+	"github.com/blevesearch/bleve/analysis/token/lowercase"
+	"github.com/blevesearch/bleve/analysis/tokenizer/unicode"
+	"github.com/blevesearch/bleve/mapping"
+	"github.com/blevesearch/bleve/registry"
+)
+
+const (
+	htmlAnalyzerName = "html"
```

Comment:
Is it a concern bleve will start having a `html` analyzer built-in? It already has a `html` char filter; I wonder why they didn't just make an analyzer. Would `kbfsHtml` be better?

Reply:
Sure, fine with me!